protocol/scripts/governance/bridge_vesters_set_rewards.py

```diff
@@ -34,13 +39,20 @@
 REWARDS_MULTIPLIER_UPDATE_2=900000 # in parts-per-million (example given: 0.9)
 # TODO: update time
 UPDATE_2_TIME_UTC="2025-12-01T15:00:00+00:00"
-# TODO: update as needed. Can be checked in Mintscan. Value range for reference: 1.2 < X < 1.35
-AVG_BLOCK_TIME_FOR_ESTIMATE=1.3
+# TODO: update as needed. Can be checked in Mintscan by visiting a future block: 
+# `www.mintscan.io/chain_name/block/9000000`
+# This can fluctuate; value range for reference: 1.2 <= X <= 1.3
+AVG_BLOCK_TIME_FOR_ESTIMATE=1.25
 ########################################################################## 
 ### END: Required proposal fields.                                     ###
 ### TODO: update above fields as needed                                ###
 ########################################################################## 
 
+##########################################################################
+### Network specific constants                                         ###
+### Only change if used on a non-prod network                          ### 
+##########################################################################
+VOTING_PERIOD_DAYS = 4
```

Comment:
nit: we should just move this under `Required proposal fields.` for simplicity

Reply:
I think this is fine for now